src/plugins/recordTypes/place/optionLists.js

```diff
@@ -107,7 +107,7 @@ export default {
       'inhabited-place',
       'island',
       'island-group',
-      'localilty',
+      'locality',
```

Comment:
This is going to change the value that is stored in the database, which means that everyone will need to do a migration to the new value. This type of change should wait for a major release of cspace (v7). Instead I would change the message key here: https://github.com/collectionspace/cspace-ui.js/blob/04c49f4a390b893b2cd701e46e9cd0a52b2f38a3/src/plugins/recordTypes/place/optionLists.js#L193

That message is not being applied right now because it doesn't match any value. If you change the message key to match the (misspelled) value, the message will be displayed for the value, and the message is correctly spelled, so it will look good to end users. The database value will still be misspelled, but there will be no data migration needed.

Reply:
Good catch.  I should have realized that since I was just merging stuff in the DB for an update to the OHC plugin.